install.sh

```diff
@@ -1847,6 +1275,15 @@ IPAM_DRIVER_EC2_SCRIPTS="src/ipamm_mad/remotes/aws/register_address_range \
 
 TM_FILES="src/tm_mad/tm_common.sh"
 
+TM_LIB_FILES="src/tm_mad/lib/kvm.rb \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

TM_LIB_FILES appears unused. Verify use (or export if used externally).